main.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"github.com/hashicorp/nomad/plugins"
 
-	pot "github.com/trivago/nomad-pot-driver/driver"
+	pot "github.com/subito/nomad-pot-driver/driver"
```

Comment:
You probably want to leave trivago in there when merging?

Reply:
you are absolutely right. I added the commit :)